cmd/openshift-install/create.go

```diff
@@ -253,6 +254,15 @@ func waitForBootstrapComplete(ctx context.Context, config *rest.Config, director
 	discovery := client.Discovery()
 
 	apiTimeout := 20 * time.Minute
+	if env, ok := os.LookupEnv("OPENSHIFT_INSTALL_BOOTSTRAP_KUBEAPI_TIMEOUT"); ok && env != "" {
+		logrus.Infof("Found override for bootstrap Kubernetes API timeout :%s", env)
+		t, err := strconv.Atoi(env)
+		if err != nil {
+			return err
+		}
+		apiTimeout = time.Duration(t) * time.Minute
+	}
```

Comment:
i'm sorry these timeout envs are not something we want to add to the installer.

Reply:
Is there any reason to use the existing hard code timeout? the quay.io may slow anywhere except in China.